test/intl402/DateTimeFormat/constructor-options-calendar-islamic-fallback.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2025 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-intl.datetimeformat
+description: >
+  Tests that fallbacks for deprecated calendars are selected from one of the
+  values returned from `AvailableCalendars`.
+info: |
+  CreateDateTimeFormat ( _newTarget_, _locales_, _options_, _required_, _defaults_ )
+
+  ...
+  9. If _resolvedCalendar_ is *"islamic"* or *"islamic-rgsa"*, then
+    a. Let _fallbackCalendar_ be an implementation- and locale-defined calendar type that is one of the values returned from AvailableCalendars.
+    b. Set _resolvedCalendar_ to CanonicalizeUValue(*"ca"*, _fallbackCalendar_).
+    c. If the ECMAScript implementation has a mechanism for reporting diagnostic warning messages, a warning should be issued.
+  10. Set _dateTimeFormat_.[[Calendar]] to _resolvedCalendar_.
+locale: [en, en-u-ca-islamic]
+features: [Intl.Era-monthcode]
+---*/
+
+const availableCalendars = [
```

Comment:
This test is incorrect. The defintion of AvailableCalendars in https://tc39.es/proposal-intl-era-monthcode/#sup-availablecalendars
is "The [List](https://tc39.es/ecma262/#sec-list-and-record-specification-type) must include the Calendar Type value of every row of [Table 1](https://tc39.es/proposal-intl-era-monthcode/#table-calendar-types), except the header row." It does said it ONLY include those listed in that Table. so the return list may also include "islamic" and "islamic-rgsa" and therefore the test is incorrect.